teslajsonpy/window_cover.py

```diff
@@ -0,0 +1,93 @@
+#  SPDX-License-Identifier: Apache-2.0
+"""
+Python Package for controlling Tesla API.
+
+For more details about this api, please refer to the documentation at
+https://github.com/zabuldon/teslajsonpy
+"""
+import time
+
+from teslajsonpy.vehicle import VehicleDevice
+
+
+class WindowCover(VehicleDevice):
+    """Home-assistant cover class for the windows of Tesla vehicles.
+
+    This is intended to be partially inherited by a Home-Assitant entity.
+    """
+
+    def __init__(self, data: dict, controller: 'teslajsonpy.Controller') -> None:  # noqa: F821
```

Comment:
Can't we just import the class above instead of suppressing the warning?

Reply:
teslajsonpy/window_cover.py:1:0: R0401: Cyclic import (teslajsonpy.controller -> teslajsonpy.window_cover) (cyclic-import)

Reply:
Oh ok.